web3/_utils/module_testing/eth_module.py

```diff
@@ -3485,8 +3527,8 @@ def test_eth_replace_transaction_legacy(
         assert replace_txn["value"] == 1
         assert replace_txn["gas"] == 21000
         assert replace_txn["gasPrice"] == txn_params["gasPrice"]
+        print("yeeeeeeehhhboiiiiiiiii")
```

Comment:
😆

Reply:
Can you tell how much joy this gave me? 😆

Reply:
lol yes I can!